src/toggl_til_tsv.js

```diff
@@ -11,14 +11,20 @@
         const tsv = time_entries_to_tsv(time_entries)
         round_to_nearest_half_hour_and_notify_if_rounding_error(tsv)
 
+
+        const locale_decimal_separator = get_locale_decimal_separator();
         return tsv.map(line =>
             line.map(col => typeof col === 'number'
-                ? col === 0 ? '' : col.toFixed(1).replace('.', ',')
+                ? col === 0 ? '' : col.toFixed(1).replace('.', locale_decimal_separator)
                 : col
             ).join('\t')
         ).join('\n')
     }
 
+    function get_locale_decimal_separator() {
+        return 1.1.toLocaleString(navigator.language).substring(1, 2)
```

Comment:
`navigator.language` _should_ be available in both FF and Chrome, although it behaves _slightly_ differently in Chrome than in Firefox (Chrome: returns UI preferred language, FF: returns first language in Accept-Language header).